align-01.html

```diff
@@ -0,0 +1,33 @@
+<style>
+
+.parent {
+    // take maximum width and height
+    size: *;
+
+    // set layout
+    flow: horizontal;
```

Comment:
the idea was not to use flow for align-01 and align-02. can you try to do it without flow?